nutils/solver.py

```diff
@@ -605,29 +605,36 @@ def __init__(self, target:types.strictstr, residual:sample.strictintegral, inert
     self.inertia = inertia
     self.theta = theta
     self.timestep = timestep
+    self.timetarget = timetarget or '_thetamethod_dummy'
```

Comment:
explain again why '_thetamethod_dummy' cannot simply be the default value for timetarget?

Reply:
I would say `timetarget` is an optional argument in the sense of `typing.Optional`. In that case `None` is a valid value. `'_thetamethod_dummy'` is an implementation detail, which shouldn't be exposed to users in any way, e.g. by introspection or by reading the docs. As an alternative to the dummy value, we could also opt for conditional replace and argument statements on lines 612 and 620. Or we could choose a default `timetarget`, say `'t'`.

Reply:
Clear, let's leave it as it is.